### PR TITLE
Profiles Sync and Databricks Destination GA

### PR DESCRIPTION
**Pull request by rchinn-segment**
### Proposed changes

Databricks GA doc updates:
- Added Databricks Destination doc to storage catalog
- Added Databricks for Profiles Sync setup doc 
### Merge timing

- launch date: 2/15
### Related issues (optional)

[DOC-776](https://segment.atlassian.net/jira/software/projects/DOC/issues/DOC-776)

[DOC-776]: https://segment.atlassian.net/browse/DOC-776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/connections/storage/catalog/databricks/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ 
+## Getting started
+
+Before getting started with the Databricks Destination, note the following prerequisites.
+
+- The target Databricks workspace must be Unity Catalog enabled. Segment doesn't support the Hive metastore. Visit the Databricks guide [enabling the Unity Catalog](https://docs.databricks.com/en/data-governance/unity-catalog/enable-workspaces.html){:target="_blank"} for more information. 
+- Segment creates [managed tables](https://docs.databricks.com/en/data-governance/unity-catalog/create-tables.html#managed-tables){:target="_blank"} in the Unity catalog. The service account needs access to create schemas on the catalog and can delete, drop, or vacuum tables.
+- Segment supports only [OAuth (M2M)](https://docs.databricks.com/en/dev-tools/auth/oauth-m2m.html){:target="_blank"}  for authentication.
+
+#### Warehouse size
```
Comment (forstisabella) [inline src/connections/storage/catalog/databricks/index.md:24]:
```suggestion
### Warehouse size
```

File: src/connections/storage/catalog/databricks/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+A SQL warehouse is required for compute. Segment recommends the following size:
```
Comment (forstisabella) [inline src/connections/storage/catalog/databricks/index.md:26]: Could also replace size with another noun that encompasses size/type/clusters
```suggestion
A SQL warehouse is required for compute. Segment recommends a warehouse with the following characteristics:
```

File: src/connections/storage/catalog/databricks/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  - **Size**: small
+  - **Type** Serverless otherwise Pro
+  - **Clusters**: Minimum of 2 - Maximum of 6
+
+> success ""
+> Segment recommends manually starting your SQL warehouse in advance. If the SQL warehouse isn't running, Segment attempts to start the SQL warehouse to validate the connection and may experience a timeout when you hit the **Test Connection** button during setup.
```
Comment (forstisabella) [inline src/connections/storage/catalog/databricks/index.md:32]:
```suggestion
> Segment recommends manually starting your SQL warehouse before setting up your Databricks destination. If the SQL warehouse isn't running, Segment attempts to start the SQL warehouse to validate the connection and may experience a timeout when you hit the **Test Connection** button during setup.
```

File: src/connections/storage/catalog/databricks/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+You'll use the Databricks workspace URL, along with Segment, to access your workspace API. 
+
+Check your browser's address bar when inside the workspace. The workspace URL should resemble: `https://<workspace-deployment-name>.cloud.databricks.com`. Remove any characters after this portion and note the URL for later use.
+
+### Step 3: Enter a Unity catalog name 
+
+This catalog is the target catalog where Segment lands your schemas and tables. 
+1. Follow the [Databricks guide for creating a catalog](https://docs.databricks.com/en/data-governance/unity-catalog/create-catalogs.html#create-a-catalog){:target="_blank"}. Be sure to select the storage location created earlier. You can use any valid catalog name (for example, "Segment"). Note this name for later use. 
+2. Select the catalog you've just created. 
+    1. Select the Permissions tab, then click **Grant**
```
Comment (forstisabella) [inline src/connections/storage/catalog/databricks/index.md:67]:
```suggestion
    1. Select the Permissions tab, then click **Grant**. 
```

File: src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+Before getting started with Databricks Profiles Sync, note the following prerequisites for setup.
+
+- The target Databricks workspace must be Unity Catalog enabled. Segment doesn't support the Hive metastore. Visit the Databricks guide [enabling the Unity Catalog](https://docs.databricks.com/en/data-governance/unity-catalog/enable-workspaces.html){:target="_blank"} for more information. 
+- Segment creates [managed tables](https://docs.databricks.com/en/data-governance/unity-catalog/create-tables.html#managed-tables){:target="_blank"} in the Unity catalog. 
+- Segment supports only [OAuth (M2M)](https://docs.databricks.com/en/dev-tools/auth/oauth-m2m.html){:target="_blank"}  for authentication.
+
+#### Warehouse size and performance 
+
+A SQL warehouse is required for compute. Segment recommends the following size:
```
Comment (forstisabella) [inline src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md:19]: Same as above - "settings" might also work, if characteristics doesn't encompass everything?

```suggestion
A SQL warehouse is required for compute. Segment recommends a warehouse with the the following characteristics:
```

File: src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+## Getting started
+
+Before getting started with Databricks Profiles Sync, note the following prerequisites for setup.
+
+- The target Databricks workspace must be Unity Catalog enabled. Segment doesn't support the Hive metastore. Visit the Databricks guide [enabling the Unity Catalog](https://docs.databricks.com/en/data-governance/unity-catalog/enable-workspaces.html){:target="_blank"} for more information. 
+- Segment creates [managed tables](https://docs.databricks.com/en/data-governance/unity-catalog/create-tables.html#managed-tables){:target="_blank"} in the Unity catalog. 
+- Segment supports only [OAuth (M2M)](https://docs.databricks.com/en/dev-tools/auth/oauth-m2m.html){:target="_blank"}  for authentication.
+
+#### Warehouse size and performance
```
Comment (forstisabella) [inline src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md:17]:
```suggestion
### Warehouse size and performance 
```

File: src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+  - **Size**: small
+  - **Type** Serverless otherwise Pro
+  - **Clusters**: Minimum of 2 - Maximum of 6
+
+
+> success ""
+> To improve the query performance of the Delta Lake, Segment recommends creating compact jobs per table using OPTIMIZE following [Databricks recommendations](https://docs.databricks.com/en/delta/optimize.html#){:target="_blank"}. 
+
+> info ""
+> Segment recommends manually starting your SQL warehouse in advance. If the SQL warehouse isn't running, Segment attempts to start the SQL warehouse to validate the connection and may experience a timeout when you hit the **Test Connection** button during setup.
```
Comment (forstisabella) [inline src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md:29]:
```suggestion
> Segment recommends manually starting your SQL warehouse before setting up your Databricks destination. If the SQL warehouse isn't running, Segment attempts to start the SQL warehouse to validate the connection and may experience a timeout when you hit the **Test Connection** button during setup.
```

File: src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+1. From your Segment app, navigate to **Unify > Profiles Sync**.
+2. Click **Add Warehouse**.
+3. Select **Databricks** as your warehouse type.
+4. Use the following steps to [connect your warehouse](#connect-your-databricks-warehouse).
+
+
+## Connect your Databricks warehouse
+
+Use the five steps below to connect your Databricks warehouse.
```
Comment (forstisabella) [inline src/unify/profiles-sync/profiles-sync-setup/databricks-profiles-sync.md:42]:
```suggestion
Use the five steps below to connect to your Databricks warehouse. 
```

File: src/connections/storage/catalog/databricks/index.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+1. Navigate to **Connections > Catalog**.
+2. Select the **Destinations** tab.
+3. Under Connection Type, select **Storage**, and click on the **Databricks storage** tile.
+4. (Optional) Select a source(s) to connect to the destination.
+5. Follow the steps below to [connect your Databricks warehouse](#connect-your-databricks-warehouse).
+
+## Connect your Databricks warehouse
+
+Use the five steps below to connect your Databricks warehouse.
```
Comment (forstisabella) [inline src/connections/storage/catalog/databricks/index.md:46]:
```suggestion
Use the five steps below to connect to your Databricks warehouse. 
```

**Review by forstisabella — APPROVED** [review]

File: src/connections/storage/catalog/databricks/index.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+    3. Click **Grant**.
+
+### Step 4: Add the SQL warehouse details from your Databricks warehouse
+
+Next, add SQL warehouse details about your compute resource. 
+- **HTTP Path**: The connection details for your SQL warehouse.
+- **Port**: The port number of your SQL warehouse.
+
+
+### Step 5: Add the service principal client ID and client secret
```
Comment (SyedWasiHaider) [inline src/connections/storage/catalog/databricks/index.md:81]: This should be OAuth Secret

File: src/connections/storage/catalog/databricks/index.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+> warning ""
+> Be sure to note the principal ID and the client secret Databricks generates, as you'll need to enter them in this step.
```
Comment (SyedWasiHaider) [inline src/connections/storage/catalog/databricks/index.md:84]: Same as above. This should say OAuth secret.

File: src/connections/storage/catalog/databricks/index.md (hunk 10 above)
Comment (rchinn-segment) [reply src/connections/storage/catalog/databricks/index.md]: updated client secret > OAuth secret

File: src/connections/storage/catalog/databricks/index.md (hunk 11 above)
Comment (rchinn-segment) [reply src/connections/storage/catalog/databricks/index.md]: updated client secret > OAuth secret

File: src/connections/storage/catalog/databricks/index.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+- Segment creates [managed tables](https://docs.databricks.com/en/data-governance/unity-catalog/create-tables.html#managed-tables){:target="_blank"} in the Unity catalog. The service account needs access to create schemas on the catalog and can delete, drop, or vacuum tables.
+- Segment supports only [OAuth (M2M)](https://docs.databricks.com/en/dev-tools/auth/oauth-m2m.html){:target="_blank"}  for authentication.
+
+> success ""
+> Segment recommends that you enable Warehouse Selective Sync. This feature enables customization of collections and properties sent to the warehouse. By syncing only relevant and required data, it reduces sync duration and compute costs, optimizing efficiency compared to syncing everything. Learn more about [Warehouse Selective Sync](/docs/connections/storage/warehouses/warehouse-syncs/#warehouse-selective-sync).
+
+### Warehouse size
+
+A SQL warehouse is required for compute. Segment recommends a warehouse with the following characteristics:
+  - **Size**: small
```
[the remark this answers is not in the document]
Comment (rchinn-segment) [reply src/connections/storage/catalog/databricks/index.md:30]: @SyedWasiHaider I've linked to the Databricks article you provided. Once Sriram's back, I can also circle back to see if he wants to update any additional sizing verbiage in this section

**Pull request merged**